package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "threatconnect-ioc-submission",
-  "version": "3.1.0",
+  "version": "3.1.2",
```

Comment:
This needs to be `3.1.1` as so the release version is increasing by one.  Don't forget to run `npm install` with node version 12 to also update the `-lock` file version as well.